spec/services/task_list_creator_spec.rb

```diff
@@ -35,11 +35,20 @@
     end
 
     it "creates actions from the workflow" do
-      section = Section.find_by(title: "Clear legal documents")
-      task = section.tasks.find_by(title: "Clear land questionnaire")
+      section = Section.find_by(title: "Starting the project")
+      task = section.tasks.find_by(title: "Understand history and complete handover from Pre-AB")
 
       expect(Action.count).to eql 6
-      expect(Action.where(title: "Action one", order: 0, task: task)).to exist
+      expect(
```

Comment:
Do you think this warrants a feature spec around helping users? Something like `Users can see hint text on Actions` and `Users can get additional guidnace on actions` - which could include tests for basic formating?